threescale_api/resources.py

```diff
@@ -608,6 +608,7 @@ def promote(self, **kwargs) -> 'Proxy':
 class Service(DefaultResource):
     AUTH_USER_KEY = "1"
     AUTH_APP_ID_KEY = "2"
```

Comment:
@mganisin what do you think about adding an `AuthTypes` inheriting from `Enum` inside `Service` resource class?

Reply:
Certainly Enum is an option